gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -110,58 +110,15 @@
 
 #define g_free_const(x) g_free((void*)(long)(x))
 
-typedef struct _TensorFilterSPList TensorFilterSPList;
-/**
- * @brief Linked list having all registered filter subplugins
- */
-struct _TensorFilterSPList
-{
-  TensorFilterSPList *next; /**< "Next" in the list */
-  GstTensorFilterFramework *body; /**< "Data" in the list element */
-};
-static GstTensorFilterFramework unknown = {
-  .name = "unknown",
-};
-static TensorFilterSPList listhead = {.next = NULL,.body = &unknown };
-
 /**
  * @brief Filter subplugin should call this to register itself
  * @param[in] tfsp Tensor-Filter Sub-Plugin to be registered
  * @return TRUE if registered. FALSE is failed or duplicated.
  */
-gboolean
+int
```

Comment:
Q. Is this for cutting the dependency on glib? In the case of tensordec_probe, it still returns gboolean.

Reply:
Yes. It's cutting dependency on glib for externally built subplugins (those who are in ext/ may choose not to use glib although our subplugins still use glib.)

The equivalent modifications for tensordec is in later PR (not the scope of this PR)